extensions/interactions/NumberWithUnits/directives/number-with-units-rules.service.ts

```diff
@@ -78,6 +84,11 @@ export class NumberWithUnitsRulesService {
     var inputsString = inputsObject.toMathjsCompatibleString();
     return unit(answerString).equals(unit(inputsString));
   }
+
+  private hasDuplicateUnits(inputsObject: any): boolean {
```

Comment:
@edensam817 Lints are failing due to explicitly using `any` type. [Error Logs](https://github.com/oppia/oppia/actions/runs/8622172652/job/23632736567?pr=20144). Please use a specific type for `inputObject`